Dockerfile

```diff
@@ -1,9 +1,9 @@
-ARG PLEX_VER=1.18.9.2578-513b381af
-ARG PLEX_SHA=8854a6aebe9ac375431b2e3e5addc7b7421dea4a
+ARG PLEX_VER=1.19.1.2589-2de7f3266
+ARG PLEX_SHA=2b7a08c41b99b40a3cd45cf687ffd3de3c61c60c
 ARG XMLSTAR_VER=1.6.1
 ARG CURL_VER=curl-7_66_0
 ARG ZLIB_VER=1.2.11
-ARG OPENSSL_VER=1.0.2u
+ARG OPENSSL_VER=1.1.1d
```

Comment:
Looking upstream, 1.1.1e is available. If this works, we should probably pull it in. It looks like it contains only bugfixes
https://www.openssl.org/news/changelog.html#openssl-111